File: MoM.Module/Dtos/SiteSettingDto.cs
namespace MoM.Module.Dtos
{
    public class SiteSettingDto
    {
        public string title { get; set; }
        public SiteSettingThemeDto theme { get; set; }
        public SiteSettingAuthenticationDto authentication { get; set; }
        public bool isInstalled { get; set; }
        public SiteSettingLogoDto logo { get; set; }
        public string modulePath { get; set; }
        public SiteSettingEmailDto email { get; set; }
    }
}

File: MoM.Web/app/dtos/SiteSettingDto.ts
import {SiteSettingThemeDto} from "./SiteSettingThemeDto";
import {SiteSettingAuthenticationDto} from "./SiteSettingAuthenticationDto";
import {SiteSettingLogoDto} from "./SiteSettingLogoDto";
import {SiteSettingEmailDto} from "./SiteSettingEmailDto";

export interface SiteSettingDto {
    title: string;
    theme: SiteSettingThemeDto;
    authentication: SiteSettingAuthenticationDto;
    isInstalled: boolean;
    logo: SiteSettingLogoDto;
    modulePath: string;
    email: SiteSettingEmailDto;
}
